src/services/workspace/WorkspaceService.js

```diff
@@ -37,6 +39,32 @@ function getAllWorkspaceFileName (organizationId, workspaceId) {
   });
 }
 
+async function fetchWorkspaceFile (organizationId, workspaceId, filePath) {
+  const accessToken = readFromStorage(ACCESS_TOKEN_LOCAL_STORAGE_KEY);
```

Comment:
Since we're using the Api config, maybe we could add a **getAccessToken** function there, instead of reading from local storage with ACCESS_TOKEN_LOCAL_STORAGE_KEY